runtime/compiler/control/CompilationThread.cpp

```diff
@@ -6267,9 +6397,61 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
             *aotCachedMethod = javaVM->sharedClassConfig->findCompiledMethodEx1(vmThread, J9_ROM_METHOD_FROM_RAM_METHOD(method), &flags);
             if (!(flags & J9SHR_AOT_METHOD_FLAG_INVALIDATED))
                {
+#ifdef COMPRESS_AOT_DATA
+               J9JITDataCacheHeader *dataCacheHeader = (J9JITDataCacheHeader*)*aotCachedMethod;
+               TR_AOTMethodHeader *aotMethodHeader = (TR_AOTMethodHeader*)(dataCacheHeader + 1);
+               if (aotMethodHeader->flags & TR_AOTMethodHeader_CompressedMethodInCache)
+                  {
+                  int sizeOfCompressedDataInCache = dataCacheHeader->size;
+                  int originalDataSize = aotMethodHeader->compileMethodDataSize + aotMethodHeader->compileMethodCodeSize;
+                  void *originalData = trMemory.allocateHeapMemory(originalDataSize);
+                  int aotMethodHeaderSize = sizeof(J9JITDataCacheHeader) + sizeof(TR_AOTMethodHeader);
+                  memcpy(originalData, *aotCachedMethod, aotMethodHeaderSize);
+                  if (inflateBuffer((U_8 *)(*aotCachedMethod)+aotMethodHeaderSize, sizeOfCompressedDataInCache-aotMethodHeaderSize, (U_8* )(originalData)+aotMethodHeaderSize, originalDataSize-aotMethodHeaderSize) == DECOMPRESSION_FAILED)
+                     {
+                     if(TR::Options::getVerboseOption(TR_VerboseAOTCompression))
```

Comment:
Formatting around `if` and parenthesis needs space to be consistent with the rest of the codebase.

Reply:
Fixed in https://github.com/eclipse/openj9/pull/5926/commits/e167856923bc9a80f32aa56be4c2152bf5a8c0bd